config/schedules/digests.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+every 10.minutes do
+  thor 'dradis:digests:send_instants'
+end
+
+every 1.day, at: '9:00' do
+  thor 'dradis:digests:send_digests'
```

Comment:
Is this the right name now?
```suggestion
  thor 'dradis:digests:send_dailies'
```

Reply:
Good catch.